SWIPs/swip-message_to_honey.md

```diff
@@ -1,23 +1,22 @@
 ---
 swip: 
 title: Message to honey oracle
```

Comment:
i would capitalize `Honey` and `Oracle` (minor gripe)

Reply:
done